torch/fx/graph.py

```diff
@@ -252,6 +252,288 @@ class _PyTreeInfo(NamedTuple):
     in_spec: pytree.TreeSpec
     out_spec: Optional[pytree.TreeSpec]
 
+@compatibility(is_backward_compatible=False)
+class CodeGen(object):
+    def __init__(self):
+        self._body_transformer: Optional[TransformCodeFunc] = None
+
+    def generate_prologue(self, free_vars: List[str], maybe_return_annotation: str) -> str:
```

Comment:
To make this consistent with the terminology used in the CPython grammar, can we name this something like `gen_fn_def`? Also concerned "prologue" might be confused with "function prologue" in another sense